From 6b7d77e52cfcedfe3431fc205975dd4fc6fcfbaf Mon Sep 17 00:00:00 2001
From: goumin <min.gou@spacemit.com>
Date: Wed, 27 Nov 2024 17:08:26 +0800
Subject: [PATCH 1310/1448] Speed up the kernel phase of Muse-book startup.

Remove earlycon=sbi, set loglevel=1, and enable asynchronous module loading with
driver_async_probe=spacemit-hdmi-drv,i2c-spacemit-k1x,ri2c-spacemit-k1x,k1xccic,sdhci-spacemit,k1x-dwc-pcie,pxa2xx-uart.

Change-Id: If6151fd331092ed76921acc6f4a704a3d2e28933
---
 arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts
index 5690722c9bed..af726c7e2766 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts
@@ -115,7 +115,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "console=ttyS0,115200n8 loglevel=1 swiotlb=65536 rdinit=/init driver_async_probe=spacemit-hdmi-drv,i2c-spacemit-k1x,ri2c-spacemit-k1x,k1xccic,sdhci-spacemit,k1x-dwc-pcie,pxa2xx-uart";
 		stdout-path = "serial0:115200n8";
 	};
 
-- 
2.47.0

